src/thor/astar_bss.cc

```diff
@@ -221,7 +221,7 @@ void AStarBSSAlgorithm::ExpandForward(GraphReader& graphreader,
     // Add to the adjacency list and edge labels.
     uint32_t idx = edgelabels_.size();
     edgelabels_.emplace_back(pred_idx, edgeid, GraphId(), directededge, newcost, sortcost, dist, mode,
-                             transition_cost, false, true, false, InternalTurn::kNoTurn,
+                             transition_cost, false, false, false, InternalTurn::kNoTurn,
```

Comment:
also this you'll see a lot: this flag is setting `closure_pruning` which is not smth we want for lots of algos (not that it matters at all, but it's again more explicit IMO to set `false` here)